service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/CompositionMapper.java

```diff
@@ -69,20 +73,22 @@ public Composition mapComposition(POCDMT000002UK01ClinicalDocument1 clinicalDocu
         }
 
 
-        if (clinicalDocument.getAuthorArray() != null) {
+        if (clinicalDocument.sizeOfAuthorArray() > 0) {
```

Comment:
it is unnecessary -> for empty array the body of foreach loop wont be called at all